configurator/src/main/java/org/picmg/configurator/StateSetTabController.java

```diff
@@ -84,30 +61,97 @@ public class StateSetTabController implements Initializable {
 	@FXML private ImageView outputUnitsImage;
 	@FXML private ImageView plusAccuracyImage;
 
+	StateSetTabController.StateSetTableData workingData = new StateSetTableData();
+	boolean modified;
+
+	public StateSetTableData getWorkingData() {
+		return workingData;
+	}
+
+
 	//TODO:change to use state sensor data class
 	//SensorTableData workingData = new SensorTableData();
 
 	//	public SensorTableData getSensorTableData() {
 	//		return workingData;
 	//	}
 	public class StateSetTableData{
+		SimpleStringProperty stateSetVendorName = new SimpleStringProperty();
+		SimpleStringProperty stateSetVendorIANA = new SimpleStringProperty();
+		SimpleStringProperty stateSetId = new SimpleStringProperty();
+		List<OEMStateValueRecord> oemStateValueRecords = new LinkedList<>();
+
+		public List<OEMStateValueRecord> getOemStateValueRecords() {
+			return oemStateValueRecords;
+		}
+
+		public void setOemStateValueRecords(List<OEMStateValueRecord> oemStateValueRecords) {
+			this.oemStateValueRecords = oemStateValueRecords;
+		}
+
+		public String getStateSetId() {
+			return stateSetId.get();
+		}
+
+		public SimpleStringProperty stateSetIdProperty() {
+			return stateSetId;
+		}
+
+		public void setStateSetId(String stateSetId) {
+			this.stateSetId.set(stateSetId);
+		}
+
+		public String getStateSetVendorIANA() {
+			return stateSetVendorIANA.get();
+		}
 
+		public SimpleStringProperty stateSetVendorIANAProperty() {
```

Comment:
Unless I am mistaken, we do not need property getters in the data object

Reply:
removed getters